landoapi/models/revisions.py

```diff
@@ -69,6 +69,9 @@ class Revision(Base):
         "LandingJob", secondary=revision_landing_job, back_populates="revisions"
     )
 
+    # A general purpose data field to store arbitrary information about this revision.
```

Comment:
What's the advantage to using a dynamic JSON object here instead of adding proper columns?

Reply:
Discussed this over zoom.